src/libs/actions/Report.js

```diff
@@ -1223,7 +1225,7 @@ function addPolicyReport(policyID, reportName, visibility, policyMembers) {
         // The room might contain all policy members so notifying always should be opt-in only.
         CONST.REPORT.NOTIFICATION_PREFERENCE.DAILY,
     );
-    const createdReportAction = ReportUtils.buildOptimisticCreatedReportAction(policyReport.ownerEmail);
+    const createdReportAction = ReportUtils.buildOptimisticCreatedReportAction(CONST.POLICY.OWNER_EMAIL_FAKE);
```

Comment:
Sorry can you clarify your question? Like why is it appropriate to set `CONST.POLICY.OWNER_EMAIL_FAKE` here?

If that's your question, here's my answer: Because previously, we were passing `CONST.REPORT.OWNER_EMAIL_FAKE,` into `ReportUtils.buildOptimisticChatReport(` above which would set that fake email as the `ownerEmail`. However, now that we're removing `policyReport.ownerEmail`, we can now just directly pass the fake email here.

However, if you're asking something like "isn't it weird to pass a FAKE email (`__FAKE__`) to `buildOptimisticCreatedReportAction` since that function will just add the fake email into the `message` property in some weird way"...

My answer is "yes that seems pretty weird, but it's how it worked before so I don't think we'll break anything with this change - and in the future it def would make sense to clean that up somehow, if possible - like maybe in this case we don't even need a `message` prop?